storage/statedb/database.go

```diff
@@ -357,6 +369,11 @@ func (db *Database) DiskDB() database.DBManager {
 	return db.diskDB
 }
 
+// TrieNodeCache retrieves the trieNodeCache of the trie database.
+func (db *Database) TrieNodeCache() *fastcache.Cache {
```

Comment:
Not related to this PR, however, it would be better if returning an interface, not a pointer of struct.

Reply:
Right. We don't have an interface for trieNodeCache now. 
Let's make a cache interface for trieNodeCache later. :)